Breeze.TumbleBit.Client/Models/Result.cs

```diff
@@ -12,27 +12,30 @@ public Result()
 	    {
 	    }
 
+        public bool CanContinue { get; private set; }
+
 		public ResultStatus Status { get; private set; }
 		public bool Success => this.Status == ResultStatus.Success; 
 
 	    public string Message { get; private set; }
 
 	    public bool Failure => !Success;
 
-	    protected Result(ResultStatus status, string message = null)
+        protected Result(ResultStatus status, string message = null, bool canContinue = true)
```

Comment:
For readability, I would change this to enum instead of a boolean. Something like `PostResultAction` with values `Stop` and `CanContinue`. It makes this line read much better without me looking up what `true` means and in the future can be extended with more action types: 

`return Result.Fail<ClassicTumblerParameters>("Not enough masternode registrations downloaded yet", true);`
compared to 
`return Result.Fail<ClassicTumblerParameters>("Not enough masternode registrations downloaded yet", PostResultAction.CanContinue);`

This is just a suggestion, I will leave it up to you to decide :)